commands/shutdown.md

```diff
@@ -1,35 +1,40 @@
 The command behavior is the following:
 
+* If there are any replicas lagging behind in replication:
+  * Pause clients attempting to write by performing a `CLIENT PAUSE` with the `WRITE` option.
+  * Wait up to the configured `shutdown-timeout` (default 10 seconds) for replicas to catch up the replication offset.
 * Stop all the clients.
 * Perform a blocking SAVE if at least one **save point** is configured.
 * Flush the Append Only File if AOF is enabled.
 * Quit the server.
 
 If persistence is enabled this commands makes sure that Redis is switched off
 without the lost of any data.
-This is not guaranteed if the client uses simply `SAVE` and then `QUIT` because
-other clients may alter the DB data between the two commands.
```

Comment:
Why did you remove this comment? Because it goes without saying and isn't related to `SHUTDOWN` or for some other reason?

Reply:
I think the comment was confusing and irrelevevant. QUIT is just closing the client's connection. It's not shutting down the server.